exercises/practice/isogram/.approaches/introduction.md

```diff
@@ -0,0 +1,64 @@
+# Introduction
+
+There are various idiomatic ways to solve Isogram.
+One approach can use a bit field to keep track of used letters.
+
+# General guidance
```

Comment:
```suggestion
## General guidance
```

Reply:
Ugh! I don't know why I keep doing that! I know better!